extensions/amp-consent/0.1/consent-config.js

```diff
@@ -206,7 +206,16 @@ export class ConsentConfig {
       // Stop at the first group that the geoService says we're in and then merge configs.
       for (let i = 0; i < geoGroups.length; i++) {
         if (geoService.isInCountryGroup(geoGroups[i]) === GEO_IN_GROUP.IN) {
-          deepMerge(mergedConfig, config['geoOverride'][geoGroups[i]], 1);
+          const geoConfig = config['geoOverride'][geoGroups[i]];
+          if (hasOwn(geoConfig, 'consentInstanceId')) {
+            user().error(
+              TAG,
+              'consentInstanceId cannot be overriden in geoGroup:',
+              geoGroups[i]
+            );
```

Comment:
Since we still merge it, should this be a user.warn() instead of error?

Reply:
Doesn't sounds like a big difference to me. I'm still not sure how are `user.warn()` get reported. But I prefer `user().error` here so that the error can be collected by analytics. 

Another option is to throw an error. I'm not sure if we want to be that strict and not fallback.

Reply:
Ok. I am fine with either, in that case